src/models/corpus.py

```diff
@@ -0,0 +1,32 @@
+"""
+Train a LDA model using mallet on the HackerNews data.
+Adapted from https://rare-technologies.com/tutorial-on-mallet-in-python/
+"""
+
+import logging
+
+from gensim import corpora, utils
+
+logging.basicConfig(format="%(asctime)s : %(levelname)s : %(message)s", level=logging.INFO)
+
+
+def iter_documents(documents):
+    """Iterate over comments, yielding one document at a time."""
+    for document in documents:
+        # parse document into a list of utf8 tokens
+        yield utils.simple_preprocess(document)
+
+
+class HackernewsCorpus(object):
```

Comment:
Can we agree on using python3? Since you can use it without the object inheritance